### DISCUSSION TITLE
HassOS e scheda wifi

### DISCUSSION THREAD

**Discussion by Hellcarrier** [indomus/forum#2377] [GENERALE - Discussioni generiche]
Buongiorno ragazzi :)
Ho installato HassOs su un minipc ed è andato tutto liscio tranne il fatto che da HA non mi vedere la scheda wifi del minipc.
Esiste un modo per fargliela riconoscere ?

**Comment by Scialla** [comment]
Questa è una cosa che non è possibile risolvere qui; devi fare delle ricerche online con la sigla o il modello del tuo minipc ed il tuo problema, ovviamente in inglese.

Io, con il mio Minix, ho risolto così, trovando online come fare
